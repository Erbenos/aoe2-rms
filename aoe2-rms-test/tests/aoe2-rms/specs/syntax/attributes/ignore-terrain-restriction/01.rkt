#lang aoe2-rms
; Place salmon on the land near each player's town center.  (Fish are can normally only be placed on water terrains)

<OBJECTS-GENERATION>
(create-object 'SALMON
               (number-of-objects 4)
               (set-place-for-every-player)
               (min-distance-to-players 3)
               (set-gaia-object-only)
               (find-closest)
               (ignore-terrain-restrictions))
