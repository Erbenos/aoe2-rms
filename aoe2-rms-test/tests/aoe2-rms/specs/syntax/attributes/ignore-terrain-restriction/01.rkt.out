<OBJECTS_GENERATION>
create_object SALMON {
  number_of_objects 4
  set_place_for_every_player
  min_distance_to_players 3
  set_gaia_object_only
  find_closest
  ignore_terrain_restrictions
}
